packages/cadl-ranch-specs/http/azure/example/basic/mockapi.ts

```diff
@@ -0,0 +1,31 @@
+import { passOnSuccess, mockapi, json } from "@azure-tools/cadl-ranch-api";
+import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+Scenarios.Client_AzureExampleClient_basicAction = passOnSuccess(
+  mockapi.post("/azure/example/basic/basic", (req) => {
```

Comment:
in this case the endpoint at scenario service is "/azure/example/basic" and we have a `@route("/azure/example/basic/basic")` decorator, the expected url should be "/azure/example/basic/azure/example/basic/basic" ?

Reply:
It actually be this. Java had this test covered.

My guess of the reason is that when there is customization (and namespace) in client.tsp, some of the decorator of e.g. `@scenarioService` on main.tsp namespace does not work (or not work the same way).

Reply:
Emm, does it cause problem on RLC when no client.tsp is read?